### ISSUE TITLE
nasty geli bug 

### ISSUE THREAD

**Issue by opntr** [HardenedBSD/hardenedBSD#169]
17:54 < Zer0P_m> nasty geli bug: 
                 http://kepfeltoltes.hu/151028/FreeBSD_www.kepfeltoltes.hu_.png
17:55 < Zer0P_m> if the disc has more than one geom object, than the geli tries each 
                 to unlock
17:56 < Zer0P_m> in this case, when you failed the password 3 times, you have (number of 
                 geom provider belongs to disc - 1) * 3 other try...

**Comment by xmj** [comment]
Did this ever get fixed?

**Comment by ColinGilbert** [comment]
No, but I want this thing fixed and am willing to work with someone to see this horror gone.
